src/twisted/web/test/test_flatten.py

```diff
@@ -632,6 +632,7 @@ def test_traceback(self) -> None:
         If a L{FlattenerError} is created with traceback frames, they are
         included in the string representation of the exception.
         """
+
```

Comment:
We can leave this change.
Was this change also generated by auto-typing ?

Maybe a bug in auto-typing and upstream might want to know about it.

Reply:
autotyping runs via libcst which automatically applies black